b288702/README.md

```diff
@@ -22,6 +22,28 @@ The following are requirements for participating in Filecoin's mainnet Phase 2.
 | Winning-PoSt-32GiB | 1GB      | 250MB          | 10s                    | 1m                    |
 | Winning-PoSt-64GiB | 1GB      | 250MB          | 10s                    | 1m                    |
 
+## Phase 1
+
+The Phase 1 output used as input to Phase 2 was derived from the [Perpetual Powers of
+Tau](https://github.com/arielgabizon/perpetualpowersoftau), by running the
+[`create_lagrange`](https://github.com/filecoin-project/powersoftau/blob/master/src/bin/create_lagrange.rs) program on
+[Challenge 19](https://github.com/arielgabizon/perpetualpowersoftau/blob/master/0018_GolemFactory_response/README.md).
+
+For this ceremony, no random beacon was used, and the `phase1radix` files used as input to phase 2 were derived
+directly from the selected challenge.
+
+> Applying the random beacon, enables proving security of the MPC under the Knowledge of Exponent assumption as shown in
+> the [paper of Bowe, Gabizon and Miers](https://eprint.iacr.org/2017/1050.pdf). However, there is no known attack when
+> the random beacon is omitted; and in fact, Mary Maller showed (see [A Proof of Security for the Sapling Generation of
+> zk-SNARK Parameters in the Generic Group
+> Model](https://github.com/zcash/sapling-security-analysis/blob/master/MaryMallerUpdated.pdf) and [Reinforcing the
+> Security of the Sapling MPC](https://electriccoin.co/blog/reinforcing-the-security-of-the-sapling-mpc/)) that the MPC
+> can be proven secure in the generic group model even when the random beacon is not used. Though the generic group
+> model is a stronger assumption than the knowledge of exponent, this is arguably not a big issue in this context, as
+> the generic group model is needed in any case for the security of the Groth16 zk-SNARK —Ariel Gabizon
```

Comment:
Is it worth turnign this into a link to Ariel's github? Or his personal website? In case someone who isn't familiar with him wants to gain confidence in his assertion

Reply:
I added a redundant link to the ceremony, which is in his GitHub repo and clarifies his credential to comment on how its output was used.